### ISSUE TITLE
Throwable class unavailable in Clojurescript

### ISSUE THREAD

**Issue by mars0i** [mikera/core.matrix#297]
Running lein figwheel on a fresh clone of aljabr modified to use core.matrix 0.55.1-SNAPSHOT, one gets the following warning on clojure/core/matrix.cljc:

```
WARNING: Use of undeclared Var clojure.core.matrix/Throwable at line 269 resources/public/js/out/clojure/core/matrix.cljc
```

**Comment by mikera** [comment]
Hmmm I suspect that I broke this inadvertantly. Any idea how we can set up ClojureScript testing?

**Comment by mars0i** [comment]
No, unfortunately.  I'm relatively new to Clojurescript--still finding surprises and working through problems.  I also have never implemented a real testing framework.  I've been more seat of the pants.  In lieu of a real testing system, running figwheel on the aljabr repo is worthwhile.  I'm trying to develop a Clojurescript core.matrix app (not working), and would be happy to be a regular informal tester to try to catch obvious issues.

**Comment by mikera** [comment]
Hmmmm I shall take a look at this. I basically need to find a way to run the ClojureScript tests bootstrapped from the Clojure testing code - maybe figwheel can do this.
